Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,24 @@
+//to deal with input.txt I had to look up how to deal with text files in Javascript
+
+//require('fs') is a node method that can be used to access files
+
+const fs = require('fs');
+//the input from input.txt
+//for some reason this is only working with the absolute path
+const textInput = fs.readFileSync('/home/serna-malala/CYF/Module-Data-Groups/Sprint-1/stretch/aoc-2018-day1/input.txt', 'utf-8');
```

Comment:
I've Updated the code so that input.txt can be accessed without the absolute path being hardcoded. Had to get the current path using node and joining it with input.txt.

This assumes that the correct directory is open